boot/freeldr/freeldr/arch/uefi/include/machuefi.h

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
I'm thinking whether those two headers need to be moved into a `boot/freeldr/freeldr/include/arch/uefi` directory instead. Especially if there are going to be more in the future.
(see https://git.reactos.org/?p=reactos.git;a=tree;f=boot/freeldr/freeldr/include/arch;h=a0ea3edc72612364ec73886562a8c2487dab0333;hb=HEAD )

Reply:
I've yet to add more headers even in the full loader branch, I'm going to do this anyway though I think it would be better.